From: Antonio Larrosa <alarrosa@suse.com>
Subject: Fix a linking to the python library

Python 3.8's python3-config doesn't seem to include the python3.8 library anymore, so
we have to add it ourselves. Use an environment variable (just as in the Windows case)
to find out the library version to link to.

Index: pgadmin4-4.19/runtime/pgAdmin4.pro
===================================================================
--- pgadmin4-4.19.orig/runtime/pgAdmin4.pro
+++ pgadmin4-4.19/runtime/pgAdmin4.pro
@@ -55,6 +55,7 @@ else {
     # Find and configure Python
     # Environment setting
     PYTHON_CONFIG = $$(PYTHON_CONFIG)
+    PYTHON_VERSION = $$(PYTHON_VERSION)
 
     # Python 2?
     isEmpty(PYTHON_CONFIG) {
@@ -71,11 +72,15 @@ else {
         error(The python-config executable could not be found. Ensure Python is installed and in the system path.)
     }
 
+    isEmpty(PYTHON_VERSION) {
+        error(Please set the PYTHON_VERSION environment variable to the version of python to link to.)
+    }
+
     message(Using $$PYTHON_CONFIG)
 
     QMAKE_CXXFLAGS += $$system($$PYTHON_CONFIG --includes)
     QMAKE_LFLAGS += $$system($$PYTHON_CONFIG --ldflags)
-    LIBS += $$system($$PYTHON_CONFIG --libs)
+    LIBS += $$system($$PYTHON_CONFIG --libs) -lpython$$PYTHON_VERSION
 
     contains( LIBS, -lpython2.* ) {
        DEFINES += PYTHON2
